igvm/hypervisor.py

```diff
@@ -285,18 +300,20 @@ def vm_set_disk_size_gib(self, vm, new_size_gib):
         """Changes disk size of a VM."""
         if new_size_gib < vm.dataset_obj['disk_size_gib']:
             raise NotImplementedError('Cannot shrink the disk.')
-        domain = self._get_domain(vm)
         with self.fabric_settings():
-            self.lvresize(self.vm_disk_path(domain.name()), new_size_gib)
+            self.lv_resize(self.vm_lv_get(vm)['path'], new_size_gib)
 
         self._vm_set_disk_size_gib(vm, new_size_gib)
 
-    def create_vm_storage(self, vm, name, transaction=None):
+    def create_vm_storage(self, vm, lv_name, transaction=None):
```

Comment:
Is it really necessary to give both VM and lv_name to create_vm_storage? Can't we move logic of choosing LV name inside this function?

Reply:
No, because we need the old lv name during online migrations and the new lv name during offline migrations and creations.